torch/csrc/distributed/rpc/rref_impl.h

```diff
@@ -257,6 +261,9 @@ class UserRRef final : public RRef {
   UserRRef(worker_id_t ownerId, const RRefId& rrefId, const ForkId& forkId);
 
   const ForkId forkId_;
+
+  // Indicates if this fork has sent delete message to it's owner.
+  std::atomic<bool> valid_{true};
```

Comment:
nit: should rename this to a more appropriate name (ex: sentDelFork and default to false).

Reply:
No problem. That's definitely more readable.